bundles/core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/Utils.java

```diff
@@ -42,4 +47,17 @@ private Utils() {}
     public static String generateId(String prefix, String path) {
         return StringUtils.join(prefix, ID_SEPARATOR, StringUtils.substring(DigestUtils.sha256Hex(path), 0, 10));
     }
-}
\ No newline at end of file
+
+    public static String generateActionURL(String modelID, String opName, String requestURI) {
```

Comment:
This looks very specific utility, its not a generic utility which can be used by all core components. Can we fix this by moving this

Reply:
Will make this private to DraftsAndSubmissions Implementation.

Reply:
Moved to another utility class under `internal.services.formsportal`